Apps/WebClient/src/ClientApp/src/store/modules/notification/mutations.ts

```diff
@@ -0,0 +1,24 @@
+import { ResultError } from "@/models/errors";
+import Notification from "@/models/notification";
+import { LoadStatus } from "@/models/storeOperations";
+
+import { NotificationMutations, NotificationState } from "./types";
+
+export const mutations: NotificationMutations = {
+    deleteNotification(state: NotificationState, notification: Notification) {
```

Comment:
Corresponding backend code has been named dismiss.  Was there a reason to not keep things consistent.